packages/perseus/src/util/graphie.test.ts

```diff
@@ -1320,6 +1321,40 @@ describe("Graphie drawing tools", () => {
             expect(onSetDrawingAreaAvailable).toHaveBeenCalledWith(false);
         });
     });
+
+    describe("getMousePx", () => {
+        it.each([
+            [{left: 0, top: 0}, {pageX: 10, pageY: 10}, [10, 10]],
+            [{top: 10, left: 20}, {pageX: 40, pageY: 40}, [20, 30]],
+        ])(
+            "should return the mouse position in pixel coordinates relative to graph",
+            (offset, mouseEvent, expectedMousePx) => {
+                const graphie = createAndInitGraphie();
+                jest.spyOn($.fn, "offset").mockReturnValue(offset);
+
+                const mousePx = graphie.getMousePx(mouseEvent);
+
+                expect(mousePx).toEqual(expectedMousePx);
+            },
+        );
+    });
+
+    describe("getMouseCoord", () => {
+        it.each([
+            [{left: 0, top: 0}, {pageX: 10, pageY: 10}, [2, 8]],
+            [{top: 10, left: 20}, {pageX: 40, pageY: 40}, [4, 4]],
```

Comment:
I'm having a hard time figuring out how the input values relate to the expected results. Could we explicitly set the scale and size of the graph, to make it a bit clearer?